plugins/polio/js/src/components/Inputs/ScopeInput.tsx

```diff
@@ -281,12 +439,81 @@ export const ScopeInput: FunctionComponent<FieldProps<Scope[], Values>> = ({
             newScopes.forEach(scope => {
                 // eslint-disable-next-line no-param-reassign
```

Comment:
Please do not disable this linter rule, it is here for a good reason: [https://eslint.org/docs/latest/rules/no-param-reassign](https://eslint.org/docs/latest/rules/no-param-reassign)